pkg/utils/kubelessutil.go

```diff
@@ -728,17 +728,26 @@ func EnsureFuncDeployment(client kubernetes.Interface, funcObj *kubelessApi.Func
 	// update deployment for loading dependencies
 	lr.UpdateDeployment(dpm, runtimeVolumeMount.MountPath, funcObj.Spec.Runtime)
 
-	livenessProbe := &v1.Probe{
-		InitialDelaySeconds: int32(3),
-		PeriodSeconds:       int32(30),
-		Handler: v1.Handler{
-			HTTPGet: &v1.HTTPGetAction{
-				Path: "/healthz",
-				Port: intstr.FromInt(int(svcPort(funcObj))),
+	livenessProbeInfo := lr.GetLivenessProbeInfo(funcObj.Spec.Runtime)
+	livenessProbe := &v1.Probe{}
+
+	if livenessProbeInfo != nil {
+		livenessProbe = livenessProbeInfo
+	} else {
```

Comment:
I would move the default livenessProbe to the function`GetLivenessProbe` just to make the code here a little bit more simple.

Reply:
Done.